src/libs/connectorLibs/saveFiles.js

```diff
@@ -301,7 +301,9 @@ async function createFile(client, entry, options, method, fileId) {
 
   let fileDocument
   if (method === 'create') {
-    fileDocument = await cozy.files.create(toCreate, createFileOptions)
+    fileDocument = await client
+      .collection('io.cozy.files')
+      .createFile(toCreate, createFileOptions)
```

Comment:
Tu devrais plutôt utiliser le save() dans ce cas. 

Pour toutes les taches « CRUD » tu ne devrais jamais utiliser la collection directement car elle bypass plein de mécanisme de Cozy client. Un jour, on pourra accéder depuis le client directement à toutes les méthodes des collections aujourd’hui seules les crud sont accessibles.

Reply:
> car elle bypass plein de mécanisme de Cozy client.

Plus spécifiquement, tu bypass notamment le store en faisant ça. Aujourd'hui on essaye de n'utiliser les collections que pour des méthodes dont on a pas accès directement depuis le client, ou pour des cas très spécifiques où on veut explicitement ne pas utiliser le store.

Reply:
Using save for creating and updating files